### PR TITLE
Don't merge or all files will be deleted from the master. I'm new to github, and had to delete the files in order to create space for the new ones. There's a 1000 file limit it seems.

### PR DESCRIPTION
**Pull request by ghost**
If you are able to merge these somehow without loosing data, that'd be great. Thanks. 

I'm trying to generate ID's and .xml in Java but I'm a new programmer and its a bit messy, I barely automated the picture resizing, thumbnail and zipping hah.

info from the commit:
________________________________________________________________________________________________
I downloaded bubbles uploaded by reddit users, merged folders manually for the same games that had slightly different names (like extra symbols) and chose best pictures of each.

Using Java I deleted all images that aren't bg0, pic0, icon0 or startup as those are the main ones and saved space. After that each image was resized to fit the guidelines and a preview image was extracted and the folders zipped.

Basically: 452 games with 4 images each + the previews

Sorry for lack of ID or template.xml

No idea how you're going to fit them in the main repository as there's a 1000 file limit.

Authors: (some games are a mishmash so better list all 4 authors in the app)

https://www.reddit.com/r/vitahacks/comments/93t3k8/sharing_bubble_arts_that_ive_made_for_use_with/

https://www.reddit.com/r/vitahacks/comments/gk33ly/my_psxpsp_official_art_faces_bubbles/

https://www.reddit.com/r/vitahacks/comments/do0a2a/the_ultimate_psp_abm_bubbles_pack_is_here_170/

https://www.reddit.com/r/vitahacks/comments/gf9hoq/i_made_some_psp_and_psx_custom_bubbles_for_abm/

### CODE REVIEW

**Comment by ONElua** [comment]
Even so, I do not know what the space limit is that github leaves me to continue hosting bubbles.
"There's a 1000 file" upsss
Maybe it's better to find a server for all the bubbles

**Comment by takuhii** [comment]
Dude, let me merge these with my pull request, I don't have a file limit

**Comment by takuhii** [comment]
> Even so, I do not know what the space limit is that github leaves me to continue hosting bubbles.
> "There's a 1000 file" upsss
> Maybe it's better to find a server for all the bubbles

BitBucket offer a 4GB repo for free, but I believe GitLab offers 10GB to it's users... So might be worth a look... I'd be happy to host the bubbles on my website if that helps...

**Comment by ONElua** [comment]
> > Even so, I do not know what the space limit is that github leaves me to continue hosting bubbles.
> > "There's a 1000 file" upsss
> > Maybe it's better to find a server for all the bubbles
> 
> BitBucket offer a 4GB repo for free, but I believe GitLab offers 10GB to it's users... So might be worth a look... I'd be happy to host the bubbles on my website if that helps...

sounds good,
We must prove that we can make direct downloads from your server in psvita

you can upload a file 
please

**Comment by takuhii** [comment]
Can you merge my feature branch if it matches your requirements please (not this one, the other PR that's still waiting), and what I will then do is upload the entire repo to my host... 

The only problem is we won't get any kind of protection like we do with GitHub, how will we make sure the files are as we specify...

**Comment by takuhii** [comment]
OK, so I cloned your repo into GitLab, it's publically viewable at the moment, the download URLs look like this;
https://gitlab.com/onelua/vitabubbles/-/raw/master/2XEXTREMEPS1RETRO.jpg?inline=false

The repo itself is here;
https://gitlab.com/onelua/vitabubbles/-/tree/master

Please be aware I have created a onelua GROUP under my account, I have not registered as ONELUA, it just makes it easier to manage users...

**Comment by ghost** [comment]
Yo guys, the file limit is only for the UI on the site. The files are still there when you clone the repository apparently. 

Or at least that were the case in 2016:
https://github.com/JuliaLang/METADATA.jl/issues/5113

**Comment by takuhii** [comment]
I've cloned the repo to gitlab and am making changes to the ABM software, I need some help from ONElua as I'm not sure how to proceed with testing.... I can take your commits and place them in a commit queue for gitlab, but I don't know how long before this is up and running.

**Comment by ghost** [comment]
Alright, fair enough.

**Comment by ONElua** [comment]
For now I do not have internet to be doing these checks

**Comment by ONElua** [comment]
Im add 3 resources
Thanks
https://github.com/ONElua/VitaBubbles/blob/master/NEWdatabase.json#L1686
